README.md

```diff
@@ -4,15 +4,30 @@
 
 ExpressJS middleware to add functionality for proxying requests to Electrum servers
 
-# Coin types
-
-BTC, LTC, DASH, ZEC
-
+## Agenda
+Web and mobile applications can't be connected with electrum servers. This package provide ability to make requests to them.
```

Comment:
Web and mobile applications can't communicate with electrum servers. This package provide ability to make requests to them.